sdk/daml-script/test/BUILD.bazel

```diff
@@ -116,7 +117,6 @@ EOF
         visibility = ["//visibility:public"],
     )
     for lf_version in [
-        "1.15",
         "1.17",
         "1.dev",
```

Comment:
Do we want to make this list of LF versions populated from daml.bzl, similar to the line

```
for version in [v for v in COMPILER_LF_VERSIONS if not (v in LEGACY_DAML_SCRIPT_LF_VERSIONS)]
```

above?